lk2nd/panel/generated/panels.h

```diff
@@ -5,6 +5,7 @@
 
 #include "lk_panel_boe_otm8019a_5p0_fwvga_video.h"
 #include "lk_panel_booyi_otm1287_720p_video.h"
+#include "lk_panel_hx8394d_720p_video.h"
```

Comment:
Do I need to rename the panel name here as well?

Reply:
No, it's fine here for now.